src/components/Fund/BudgetDetails/BudgetDetails.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { MultiColumnList } from '@folio/stripes/components';
+import { AmountWithCurrencyField } from '@folio/stripes-acq-components';
+
+const visibleColumns = ['name', 'code', 'allocated', 'unavailable', 'available'];
+const columnMapping = {
+  name: <FormattedMessage id="ui-finance.fund.budget.name" />,
+  code: <FormattedMessage id="ui-finance.fund.budget.code" />,
+  allocated: <FormattedMessage id="ui-finance.budget.allocated" />,
+  unavailable: <FormattedMessage id="ui-finance.budget.unavailable" />,
+  available: <FormattedMessage id="ui-finance.budget.available" />,
+};
+const columnWidths = {
+  name: '20%',
+  code: '20%',
+  allocated: '20%',
+  unavailable: '20%',
+  available: '20%',
+};
+
+const BudgetDetails = ({ budgets, currency, openBudget }) => {
+  const resultsFormatter = {
```

Comment:
imo formatter can be memoized based on currency

Reply:
imo not important here